contrib/macdeploy/macdeployqtplus

```diff
@@ -157,20 +157,21 @@ class DeploymentInfo(object):
         self.qtPath = None
         self.pluginPath = None
         self.deployedFrameworks = []
-    
+
     def detectQtPath(self, frameworkDirectory: str):
         parentDir = os.path.dirname(frameworkDirectory)
-        if os.path.exists(os.path.join(parentDir, "translations")):
-            # Classic layout, e.g. "/usr/local/Trolltech/Qt-4.x.x"
+        if os.path.exists(os.path.join(parentDir, "translations")) or os.path.exists(os.path.join(parentDir, "share", "qt", "translations")):
```

Comment:
In f7bd54d5f34361c0c06dc86fbd8e2497e7e4473c: Why are we adding support for 6, while retaining support for 5, if we now require 6?

Reply:
Thanks! Adjusted.